lib/core/src/Cardano/Wallet/Primitive/Model.hs

```diff
@@ -555,20 +583,32 @@ applyOurTxToUTxO
     -> s
     -> Tx
     -> UTxO
-    -> Maybe ((Tx, TxMeta), UTxO)
-applyOurTxToUTxO !slotNo !blockHeight !s !tx !prevUTxO =
+    -> Maybe ((Tx, TxMeta), DeltaUTxO, UTxO)
+applyOurTxToUTxO !slotNo !blockHeight !s !tx !u0 =
     let
         -- The next UTxO state (apply a state transition) (e.g. remove
         -- transaction outputs we've spent)
-        ourNextUTxO =
-            spendTx tx prevUTxO
-            <> UTxO.filterByAddress (ours s) (utxoFromTx tx)
+        (du, u) = (du21 <> du10, u2)
+
+        (du10, u1)   = spendTxD tx u0
```

Comment:
Maybe it would be good to add comment why du21 and du10 has this naming

Reply:
I have added a comment